corrector.rb

```diff
@@ -0,0 +1,11 @@
+class Corrector
+  attr_accessor :name
+
+  def correct_name(name)
+    if name.length > 10
+      name.capitalize[0..9]
+    else
+      name.capitalize
+    end
```

Comment:
- [ ] in order to make this work you need to add the exclamation mark (!) behind the `capitalize` method otherwise, it will not change the actual variable.